server/app/models/grid_service.rb

```diff
@@ -67,8 +67,12 @@ def to_path
   end
 
   # @return [String]
-  def name_with_stack
-    "#{self.stack.name}-#{self.name}"
+  def name_with_stack(ignore_default_stack: false, stack_separator: '-')
```

Comment:
Yeah, dunno what the exact name is, but if it's `/`-separated, then it should be some kind of "path"?

* Docker container names are `-`-separated
* DNS names are `.` -separated
* API names are `/` -separated